coverage.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
gitignore?

Reply:
why? We need it for actions

Reply:
File like that will be generated during the action, won't it?

Reply:
aww! yep